Formula/d/docker-compose.rb

```diff
@@ -6,6 +6,11 @@ class DockerCompose < Formula
   license "Apache-2.0"
   head "https://github.com/docker/compose.git", branch: "main"
 
+  livecheck do
```

Comment:
```suggestion
  # Upstream creates releases that use a stable tag (e.g., `v1.2.3`) but are
  # labeled as "pre-release" on GitHub before the version is released, so it's
  # necessary to use the `GithubLatest` strategy.
  livecheck do
```

The newest release uses a one-off tag format (`v2.24.0-birthday.10` instead of the usual `v2.23.3`) but this is the general idea, as `v2.24.0-birthday.10` is currently marked as "pre-release".